api/src/main/java/marquez/db/LineageDao.java

```diff
@@ -25,8 +28,19 @@
 @RegisterRowMapper(JobDataMapper.class)
 @RegisterRowMapper(RunMapper.class)
 @RegisterRowMapper(JobRowMapper.class)
+@RegisterRowMapper(SimpleLineageEdgeMapper.class)
 public interface LineageDao {
 
+  public record SimpleLineage(Collection<SimpleLineageEdge> edges) {}
```

Comment:
Instead of `SimpleLineage`, I would prefer `DirectLineage` or `AdjacentLineage` (as adjacent nodes in graph)

Reply:
Ah yes, I renamed the endpoint and then forgot to do it everywhere.
Do you prefer `direct` or `adjacent`?

Reply:
Both are fine.

Reply:
updated with `direct`